apps/ima/views/declaration-person-named.html

```diff
@@ -0,0 +1,15 @@
+{{<partials-page}} {{$page-content}}
+ {{#markdown}}declaration-person-named{{/markdown}}
+ <br><br>
+ <div class="govuk-checkboxes__item">
+    <input class="govuk-checkboxes__input" id="declaration-person-named" name="declarationPersonNamed" type="checkbox">
+    <label class="govuk-label govuk-checkboxes__label" for="declaration-person-named">
+      {{#t}}fields.declaration-person-named.label{{/t}}
+    </label>
+  </div>
+  <br><br>
+  <p>You will not be able to change your answer after you submit.</p>
```

Comment:
Should be 'You will not be able to change your answers after you submit'.

Reply:
Changed to answers